kpi/deployment_backends/kobocat_backend.py

```diff
@@ -610,12 +618,16 @@ def __get_submissions_in_xml(self, instance_ids=[], **kwargs):
         :return: list<XML>
         """
 
-        sort = {'id': 1}
         kwargs['instance_ids'] = instance_ids
 
         if 'fields' in kwargs:
             raise ValueError(_('`Fields` param is not supported with XML format'))
 
+        # FIXME. Use Mongo to sort data and ask PostgreSQL to follow the order.
+        # See. https://stackoverflow.com/a/867578
+        if 'sort' in kwargs:
+            raise ValueError(_('`sort` param is not supported with XML format'))
```

Comment:
The problem with `ValueError` here is that it will return a 500 to the client, and with `DEBUG=False`, the explanatory message will be hidden.
![image](https://user-images.githubusercontent.com/2085013/64381416-21ac5e80-d001-11e9-83a1-a7a6702b4cf2.png)